tests/MongoDB.Analyzer.Tests/MongoDB.Analyzer.Tests.csproj

```diff
@@ -22,12 +22,20 @@
         <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
     <PackageReference Include="System.Text.Json" Version="5.0.2" />
+    <PackageReference Include="MongoDB.Bson" Version="2.16.1" />
```

Comment:
Do we need BSON?

Reply:
When Building the test project, the test cases ran perfectly but a message appeared saying that the BSON assembly couldn't be located. Didn't impact the test run at all. I removed this package reference